tap_mailchimp_export/streams.py

```diff
@@ -201,13 +212,18 @@ def run_export_request(ctx, entity, stream, last_updated, retries=0):
             ) as res:
                 if stream == IDS.CAMPAIGN_SUBSCRIBER_ACTIVITY:
                     batched_records = \
-                        handle_campaign_subscriber_activity_response(
+                        handle_subscriber_activity_response(
                             res, stream, entity, last_updated
                         )
                 elif stream == IDS.LIST_MEMBERS:
                     batched_records = handle_list_members_response(
                         res, stream, entity, last_updated
                     )
+                elif stream == IDS.AUTOMATION_WORKFLOW_SUBSCRIBER_ACTIVITY:
```

Comment:
can make this drier by addin as `if stream in IDS.CAMPAIGN_SUBSCRIBER_ACTIVITY,  IDS.AUTOMATION_WORKFLOW_SUBSCRIBER_ACTIVITY] `